e2e/newenemy/newenemy_test.go

```diff
@@ -128,6 +128,9 @@ func TestNoNewEnemy(t *testing.T) {
 	require.NoError(t, vulnerableSpiceDb.Start(ctx, tlog, "vulnerable",
 		"--datastore-tx-overlap-strategy=insecure"))
 	require.NoError(t, vulnerableSpiceDb.Connect(ctx, tlog))
+	t.Cleanup(func() {
+		require.NoError(t, vulnerableSpiceDb.Stop(tlog))
```

Comment:
If one of the cleanup funcs fails the require check, does that prevent subsequent cleanup funcs from running?

Reply:
All of the registered cleanup funcs should run: https://cs.opensource.google/go/go/+/refs/tags/go1.17.6:src/testing/testing.go;l=1019;drc=refs%2Ftags%2Fgo1.17.6;bpv=1;bpt=1